src/main/java/org/apache/commons/io/build/AbstractOrigin.java

```diff
@@ -267,7 +521,7 @@ public FileOrigin(final File origin) {
 
         @Override
         public byte[] getByteArray(final long position, final int length) throws IOException {
-            try (RandomAccessFile raf = RandomAccessFileMode.READ_ONLY.create(origin)) {
+            try (RandomAccessFile raf = RandomAccessFileMode.READ_ONLY.create(getFile())) {
```

Comment:
How is this change related to Channels? This is a giant PR already. Can it be focused on just the new feature? Or is this a bug fix?

Reply:
Fixed in https://github.com/apache/commons-io/pull/784/commits/24fff74dc1645e7b566fb167befe1ad167c1ba62